### PR TITLE
Cinema project implemented

### PR DESCRIPTION
**Pull request by thorium-zero**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self._number = number
+
+    def movie_session(
+            self, movie_name: str,
+            customers: list[Customer],
+            cleaner: Cleaner) -> None:
```
Comment (YuriiKindrat) [inline app/cinema/hall.py:12]: fix indentation here, please
move your closing bracket to the new line or delete a blank line after opening bracket

File: app/cinema/hall.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        print(f'"{movie_name}" started in hall number {self._number}.')
+        for cust in customers:
```
Comment (YuriiKindrat) [inline app/cinema/hall.py:14]: maybe it is better to name this variable as customer ?)

File: app/cinema/hall.py (hunk 1 above)
Comment (YuriiKindrat) [reply app/cinema/hall.py:12]: same is for similar cases

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list,
+        hall_number: int,
+        cleaner: str,
+        movie: str) -> None:
+    customers_list: list[Customer] = \
+        [Customer(current["name"], current["food"]) for current in customers]
```
Comment (YuriiKindrat) [inline app/main.py:13]: no need to use linebreaker here, u can break a line through the brackets

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done 😎🎉

**Pull request closed without merging**